core/domain/collection_services.py

```diff
@@ -479,13 +480,18 @@ def get_collection_ids_matching_query(query_string, cursor=None):
 
     Args:
         query_string: str. The search query string.
+        cursor: Cursor or None. Cursor pointing to the start of the collection
```

Comment:
I traced through the search function code to gae_search_services.py, and the cursor is a string, my apologies.